2022-05-20 12:26:47,266 INFO  ==============================================================
2022-05-20 12:26:47,267 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-05-20 12:26:47,268 INFO  Dir:   software_management_end_to_end/
2022-05-20 12:26:47,268 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-05-20 12:26:47,269 INFO  ==============================================================
2022-05-20 12:26:47,279 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-05-20 12:26:47,280 INFO  EnvironmentC8y Setup
2022-05-20 12:26:47,842 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-05-20 12:26:47,952 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 4 seconds
2022-05-20 12:26:47,954 INFO  Delaying execution by 7 seconds
2022-05-20 12:27:09,147 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 14 secs
2022-05-20 12:27:09,764 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-05-20 12:27:10,228 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-05-20 12:27:10,690 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-05-20 12:27:12,915 INFO  State of current operation: SUCCESSFUL
2022-05-20 12:27:13,138 INFO  Response status: 201
2022-05-20 12:27:13,140 INFO  Response to action: {
    "creationTime": "2022-05-20T12:27:13.090Z",
    "deviceId": "19392643",
    "deviceName": "m32sd10a",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409548",
    "id": "19409548",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-05-20 12:27:13,141 INFO  Started operation: {'creationTime': '2022-05-20T12:27:13.090Z', 'deviceId': '19392643', 'deviceName': 'm32sd10a', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409548', 'id': '19409548', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-05-20 12:27:15,359 INFO  State of operation 19409548 : PENDING
2022-05-20 12:27:17,571 INFO  State of operation 19409548 : EXECUTING
2022-05-20 12:27:19,781 INFO  State of operation 19409548 : EXECUTING
2022-05-20 12:27:22,004 INFO  State of operation 19409548 : EXECUTING
2022-05-20 12:27:24,219 INFO  State of operation 19409548 : FAILED
2022-05-20 12:27:24,220 ERROR Stopping as the operation has failed
2022-05-20 12:27:24,221 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-05-20 12:27:24,243 WARN  SystemError: The operation has failed ... blocked 
2022-05-20 12:27:24,245 INFO  
2022-05-20 12:27:24,245 INFO  cleanup:
2022-05-20 12:27:28,264 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-05-20 12:27:28,833 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-05-20 12:27:28,837 INFO  
2022-05-20 12:27:28,837 INFO  Test duration: 41.58 secs
2022-05-20 12:27:28,838 INFO  Test final outcome:  BLOCKED
2022-05-20 12:27:28,839 INFO  Test outcome reason: SystemError: The operation has failed
2022-05-20 12:27:28,840 INFO  
